[EventGrid] Read me + improve docstrings (#13484)
Azure/azure-sdk-for-python: sdk/eventgrid/azure-eventgrid/CHANGELOG.md

# Release History

## 2.0.0b1 (2020-09-08)

  - Version (2.0.0b1) is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure EventGrid.
  
  For more information about this, and preview releases of other Azure SDK libraries, please visit https://azure.github.io/azure-sdk/releases/latest/python.html.

>   **Breaking Changes**

Reviewer: So looking at what you put under here, I'm not sure it's set up right.  I'd probably put these under **Features** (and put that header above your top-level preview effort snippet), and if you wanted a breaking changes section (which I thought was reasonable) you could mention the deprecation of the track-1 publishing method?

>   - Implements the `EventGridPublisherCloent` for the publish flow for EventGrid Events, CloudEvents and CustomEvents.

Reviewer: ```suggestion
  - Implements the `EventGridPublisherClient` for the publish flow for EventGrid Events, CloudEvents and CustomEvents.
```

> ## 2.0.0b1 (Unreleased)

Reviewer: ```suggestion
## 2.0.0b1 (2020-09-08)
```

Author: fixeed

>   - Version (2.0.0b1) is the first preview of our efforts to create a user-friendly and Pythonic client library for AzureEventGrid.

Author: ```suggestion
  - Version (2.0.0b1) is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure EventGrid.
```

>   - Version (2.0.0b1) is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure EventGrid.

Reviewer: Nit:  Might add some Very High Level notes like "implements basic publish and consume flows for EventGridEvents, CloudEvents, and CustomEvents, as well as model types for system events."

Author: added **breaking changes**
